dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp

```diff
@@ -874,33 +873,26 @@ void DeltaMergeStore::ingestFiles(
         // Example: "ingest dmf_1001,1002,1003 into segment [1,3]"
         //          "ingest <empty> into segment [1,3]"
         FmtBuffer fmt_buf;
-        if (file_ids.empty())
-        {
-            fmt_buf.append("ingest <empty>");
-        }
-        else
-        {
-            fmt_buf.append("ingest dmf_");
-            fmt_buf.joinStr(
-                file_ids.begin(),
-                file_ids.end(),
-                [](const PageId id, FmtBuffer & fb) { fb.fmtAppend("{}", id); },
-                ",");
-        }
-        fmt_buf.append(" into segment [");
+        fmt_buf.append("ingested_files=[");
+        fmt_buf.joinStr(
+            file_ids.begin(),
+            file_ids.end(),
+            [](const PageId id, FmtBuffer & fb) { fb.fmtAppend("dmf_{}", id); },
+            ",");
+        fmt_buf.append("] updated_segment=[");
         fmt_buf.joinStr(
             updated_segments.begin(),
             updated_segments.end(),
-            [](const auto & segment, FmtBuffer & fb) { fb.fmtAppend("{}", segment->segmentId()); },
+            [](const auto & segment, FmtBuffer & fb) { fb.fmtAppend("{}", segment->simpleInfo()); },
             ",");
         fmt_buf.append("]");
+
         LOG_FMT_INFO(
             log,
-            "table: {}.{}, clear_data: {}, {}",
-            db_name,
-            table_name,
+            "Finish table ingest files, ingested files into segments, {} clear={} store={}",
+            fmt_buf.toString(),
```

Comment:
we can use a lambda function to avoid generating the `FmtBuffer` if the log level is warning or higher, like
```
auto get_ingest_info = [&]() -> String {
  FmtBuffer fmt_buf;
  fmt_buf.append(...
  ...
  return fmt_buf.toString();
};
LOG_FMT_INFO(
  log,
  "Finish table ingest files, ingested files into segments, {} clear={} store={}",
  get_ingest_info(),
  clear_data_in_range,
  simpleInfo());
```